src/scripts/utils/toast.ts

```diff
@@ -0,0 +1,44 @@
+import StartToastifyInstance from "toastify-js";
+import Toastify from "toastify-js";
+import "../../styles/toast.css";
+
+interface ToastOptions extends StartToastifyInstance.Options {
+  description: string;
+  title?: string | HTMLElement;
+}
+
+export function showToast(options: ToastOptions) {
+  const { description, avatar, title } = options;
+  const toastNode = document.createElement("div");
+  const toastHeader = document.createElement("div");
+  const toastMessage = document.createElement("div");
+
+  toastHeader.style.fontWeight = "bold";
+  toastHeader.style.fontSize = "20px";
+  toastHeader.style.color = "black";
+  toastHeader.innerHTML = `<div class="toast-header">${
+    avatar ? `<img src="${avatar}">` : ""
+  }<span>${title}</span></div>`;
```

Comment:
Does this syntax actually allow `title: HTMLElement`?

Reply:
@pkong-ds I'm not sure which better, how about 
```
title: string | HTMLElement
```

use default `<div class="toast-header">.....</div>` if it' string, and directly use `title` if it's HTMLElement

Reply:
I was thinking simply type-forcing it as string, i.e.

```
title: string
```

Why do you want yo support HTMLElement here? What usage do you expect?

Reply:
I think `string | HTMLElement` is more flexible, simple string ok too.

Reply:
ok! let's go for what you suggested~

> use default `<div class="toast-header">.....</div>` if it' string, and directly use `title` if it's HTMLElement